training/src/anemoi/training/train/tasks/rollout.py

```diff
@@ -82,6 +82,25 @@ def __init__(
         LOGGER.debug("Rollout increase every : %d epochs", self.rollout_epoch_increment)
         LOGGER.debug("Rollout max : %d", self.rollout_max)
 
+    def _get_input(self, batch: torch.Tensor) -> torch.Tensor:
+        """Get input tensor shape for diffusion model."""
+        x = batch[
+            :,
+            0 : self.multi_step,
+            ...,
+            self.data_indices.data.input.full,
+        ]  # (bs, multi_step, ens, latlon, nvar)
+        msg = f"Batch length not sufficient for requested multi_step length!, {batch.shape[1]} !>= {self.multi_step}"
```

Comment:
is this the same error message as line 57 in forecaster.py? do we need to keep the two?